test/server_integration/http/ssl_redirect/config.js

```diff
@@ -38,20 +37,18 @@ export default async function ({ readConfigFile }) {
   const httpConfig = await readConfigFile(require.resolve('../../config'));
   const certificateAuthorities = [readFileSync(CA_CERT_PATH)];
 
-  const redirectPort = httpConfig.get('servers.opensearchDashboards.port') + 1234;
+  const opensearchDashboardsUrl = new URL('', httpConfig.get('servers.opensearchDashboards'));
```

Comment:
Is `httpConfig.get('servers.opensearchDashboards')` not already a `URL`?

Reply:
It's a `URL`, and we need to clone a new `URL` here. Because we need to change the port later. I think we can remove the force validation here.